src/Sylius/Behat/Context/Ui/Shop/AccountContext.php

```diff
@@ -423,4 +423,12 @@ public function iWantToLogIn()
     {
         $this->loginPage->tryToOpen();
     }
+    
+    /**
+     * @Then The order's shipment status should be :shippingStatus
+     */
+    public function theOrderSShipmentStatusShouldBe(string $shippingStatus): void
+    {
+        Assert::same($this->orderShowPage->getShippingStatus(), $shippingStatus);
```

Comment:
We have inconsistency in the naming, here it is a **shipping status** but in step, it is a **shipment status** 🤷‍♂️ We should choose only one name :)